docs/api/index-api.rst

```diff
@@ -234,6 +236,8 @@ Other
     count()           <dt/count>
     countna()         <dt/countna>
     cov()             <dt/cov>
+    cummin()          <dt/cummin>
+    cummax()          <dt/cummax>
```

Comment:
I guess I put it in the wrong order here, `cummax()` should go first alphabetically. May be you could change the order here and below.